pytest.ini

```diff
@@ -1,2 +1,14 @@
 [pytest]
 addopts = --snapshot-patch-pycharm-diff
+# this shouldn't be necessary since it's already in pyproject.toml
+# but tests show lots of warnings without it
+markers =
```

Comment:
the markers are in pyproject.toml, this file shouldn't exist (deleting it on main now)

Reply:
This file was introduced recently by mistake, this is why you encountered 

> Adds the custom marker config to pytest.ini: in contradiction with the claim in pytest docs, tests were throwing warnings about unknown custom marks 🤷🏽